docker/Dockerfile.hipcc

```diff
@@ -1,8 +1,11 @@
-ARG BASE=rocm/dev-ubuntu-18.04:3.5
+ARG BASE=rocm/dev-ubuntu-20.04:3.8
 FROM $BASE
 
 ARG NPROCS=4
 
+# Avoid tzdata dialog
+ARG DEBIAN_FRONTEND=noninteractive
```

Comment:
No.  https://docs.docker.com/engine/faq/#why-is-debian_frontendnoninteractive-discouraged-in-dockerfiles
Use `DEBIAN_FRONTEND=noninteractive && apt-get update && ...` if you really have to.

Reply:
Very nice catch that I did not think of.

Reply:
Apparently, this doesn't work as intended. Since this is `ARG` and not `ENV` it should be fine anyway, don't you agree?

Reply:
`RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -yq ...`

Reply:
Here you go.